book/tutorials_examples/1_HBV_Caravan_ERA5/example_model_run_HBV.ipynb

```diff
@@ -34,7 +34,7 @@
     "The stucture of the HBV model is shown below.\n",
```

Comment:
<p>change reference of </p><p> </p><ul><li>"where we are in",</li><li>"these are more computationally intensive to run and therefore unsuited for a short 45 minute workshop."</li></ul><p> </p><p> because no longer part of workshop.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/projects/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>